plugins/USBPrinting/__init__.py

```diff
@@ -19,5 +19,5 @@ def getMetaData():
     }
 
 def register(app):
-    qmlRegisterSingletonType(USBPrinterManager.USBPrinterManager, "UM", 1, 0, "USBPrinterManager", USBPrinterManager.USBPrinterManager.getInstance)
-    return {"extension":USBPrinterManager.USBPrinterManager.getInstance(),"output_device": USBPrinterManager.USBPrinterManager.getInstance() }
+    qmlRegisterSingletonType(USBPrinterOutputDeviceManager.USBPrinterOutputDeviceManager, "Cura", 1, 0, "USBPrinterManager", USBPrinterOutputDeviceManager.USBPrinterOutputDeviceManager.getInstance)
```

Comment:
Come to think of it, the Qt documentation for qmlRegisterSingletonType states:

> NOTE: A QObject singleton type instance returned from a singleton type provider is owned by the QML engine. 
> For this reason, the singleton type provider function should not be implemented as a singleton factory.

This is another one of the reasons for using proxy objects in Uranium. I expect it never was a problem because of how Python and PyQt handle objects, but it is something to be aware of. I do not currently know an easy solution, but please do add a comment here that we are basically violating QML api. :)